webapp/src/main/java/com/box/l10n/mojito/rest/scheduledjob/ScheduledJobWS.java

```diff
@@ -35,18 +39,159 @@ public class ScheduledJobWS {
   static Logger logger = LoggerFactory.getLogger(ScheduledJobWS.class);
   private final ScheduledJobRepository scheduledJobRepository;
   private final ScheduledJobManager scheduledJobManager;
+  private final ScheduledJobStatusRepository scheduledJobStatusRepository;
+  private final ScheduledJobTypeRepository scheduledJobTypeRepository;
 
   @Autowired
   public ScheduledJobWS(
-      ScheduledJobRepository scheduledJobRepository, ScheduledJobManager scheduledJobManager) {
+      ScheduledJobRepository scheduledJobRepository,
+      ScheduledJobManager scheduledJobManager,
+      ScheduledJobStatusRepository scheduledJobStatusRepository,
+      ScheduledJobTypeRepository scheduledJobTypeRepository) {
     this.scheduledJobRepository = scheduledJobRepository;
     this.scheduledJobManager = scheduledJobManager;
+    this.scheduledJobStatusRepository = scheduledJobStatusRepository;
+    this.scheduledJobTypeRepository = scheduledJobTypeRepository;
   }
 
   private final ResponseEntity<ScheduledJobResponse> notFoundResponse =
       createResponse(
           HttpStatus.NOT_FOUND, ScheduledJobResponse.Status.FAILURE, "Job doesn't exist", null);
 
+  @RequestMapping(method = RequestMethod.POST, value = "/api/jobs/create")
+  @ResponseStatus(HttpStatus.OK)
+  public ResponseEntity<ScheduledJobResponse> createJob(@RequestBody ScheduledJob scheduledJob) {
```

Comment:
```suggestion
  public ScheduledJob createJob(@RequestBody ScheduledJob scheduledJob) {
```

We should let this endpoint return just the ScheduledJob that was created. 
The ScheduledJobResponse was used for responding specifically to trigger and enable/disable events.

For returning a `BAD_REQUEST` we can throw a `ResponseStatusException` example:

```
new ResponseStatusException(HttpStatus. BAD_REQUEST, "Repository must be provided to create a job")